pkg/volume/volume.go

```diff
@@ -237,7 +238,7 @@ type Deleter interface {
 	// sync. This can be used to postpone deletion of a volume that is being
 	// detached from a node. Deletion of such volume would fail anyway and such
 	// error would confuse users.
-	Delete() error
+	Delete(logger klog.Logger) error
```

Comment:
because of here, it uses klog.V(4).Info
https://github.com/kubernetes/kubernetes/blob/88cce39155f3080cfe3163c9afb39b7572ca645c/pkg/controller/volume/persistentvolume/framework_test.go#L1052
i will rollback it

Reply:
done